judge/judgedaemon.main.php

```diff
@@ -838,14 +842,16 @@ function fetch_executable_internal(
                     $run_config['hash']
                 );
                 if (isset($error)) {
-                    // FIXME
-                    continue;
+                    $scriptId = $judgeTask['run_script_id'];
+                    disable('judgehost', 'hostname', $myhost, "Retrieving/storing debug script '$scriptId' failed.");
```

Comment:
This should disable the script not the judgehost. I wonder why we are not calling `fetch_executable` above (instead of the `_internal` variant which would do this for us if I am not mistaken).